course_discovery/apps/course_metadata/models.py

```diff
@@ -1739,13 +1739,16 @@ def get_seat_upgrade_deadline(self, seat_type):
                 deadline = subtract_deadline_delta(self.end, settings.PUBLISHER_UPGRADE_DEADLINE_DAYS)
         return deadline
 
-    def update_or_create_seat_helper(self, seat_type, prices):
+    def update_or_create_seat_helper(self, seat_type, prices, upgrade_deadline_override):
         defaults = {
             'upgrade_deadline': self.get_seat_upgrade_deadline(seat_type),
         }
         if seat_type.slug in prices:
             defaults['price'] = prices[seat_type.slug]
 
+        if upgrade_deadline_override and seat_type.slug != Seat.AUDIT:
```

Comment:
I guess the upgrade deadline is only meant for Verified seat types and we can update this condition like this:
```suggestion
        if upgrade_deadline_override and seat_type.slug != Seat.VERIFIED:
```

Reply:
if upgrade deadline is only meant for Verified seat types then condition should be like 
`if upgrade_deadline_override and seat_type.slug == Seat.VERIFIED:`